charms/istio-pilot/src/charm.py

```diff
@@ -42,6 +43,13 @@
     "recommendations."
 )
 
+# Helper to retry calling a function over 300 seconds
+RETRY_FOR_15_MINUTES = tenacity.Retrying(
+    stop=tenacity.stop_after_delay(60 * 15),
```

Comment:
60*5=300 
60*15=900

:D

Reply:
haha this is a typo in the comment.  Before it was 300, but bumped to 1500 just in case.  Fixed the comment